Routing/Loader/RestXmlCollectionLoader.php

```diff
@@ -70,6 +70,7 @@ protected function parseNode(RouteCollection $collection, \DOMElement $node, $pa
                 $namePrefix = (string) $node->getAttribute('name-prefix');
                 $parent = (string) $node->getAttribute('parent');
                 $type = (string) $node->getAttribute('type');
+                $host = isset($config['host']) ? $config['host'] : null;
```

Comment:
I agree with @stloyd I don't think we need this variable. Maybe we should remove the other variables as well.